src/components/WorkNumberSelect.tsx

```diff
@@ -85,12 +85,15 @@ export default function WorkNumberSelect({
       {optionValues(works, "workNumber", "workNumber")}
     </FormikSelect>
   ) : (
+    <>
     <Select
       value={selectedWorkNumber}
       onChange={handleWorkNumberChange}
       emptyOption={true}
     >
       {optionValues(works, "workNumber", "workNumber")}
     </Select>
+    { selectedWorkNumber === "" ? <p className="text-red-500 text-xs italic">SGP number is required</p> : "" }
```

Comment:
Not sure if there is a build in class for validation / error messages in non formik components that could replace this?